### PR TITLE
#3188- Autotests: Bond tool

### PR DESCRIPTION
**Pull request by AKZhuk**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts
Comment (SashaGraves) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts:89]: are we sure we need this 'only'? 
As from documentation:
"When there are focused with "only" tests, only these tests run in entire project."

**Review by SashaGraves — APPROVED** [review]

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts
Comment (AKZhuk) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts:89]: Thank you for noticed. I removed `only`

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts
Comment (y-holik) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts:392]: there's a function which allows you to open any dropdown on left panel toolbar (so you don't have to do double clicks)
image

Comment (AKZhuk) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Bond-Tool/all-bonds.spec.ts:392]: I believe it is redundant work to create function twice. I guess we should wait until your changes will merge and use this function in the future

**Review by y-holik — APPROVED** [review]

**Pull request merged**
